projects/tinkerbell/cluster-api-provider-tinkerbell/patches/0015-Make-hardware-template-workflow-CRDs-namespace-scope.patch

```diff
@@ -0,0 +1,173 @@
+From 0af1ad2983f231b3c41a44cb3414f5b3f3d623e4 Mon Sep 17 00:00:00 2001
+From: Pooja Trivedi <tripooja@amazon.com>
+Date: Mon, 2 May 2022 11:52:40 -0700
+Subject: [PATCH] Make hardware, template, workflow CRDs namespace-scoped
+
+---
+ config/crd/bases/tinkerbell.org_hardware.yaml  | 2 +-
+ config/crd/bases/tinkerbell.org_templates.yaml | 2 +-
+ config/crd/bases/tinkerbell.org_workflows.yaml | 2 +-
+ config/default/manager_image_patch.yaml        | 2 +-
+ config/default/manager_pull_policy.yaml        | 2 +-
+ controllers/base.go                            | 2 ++
+ controllers/machine.go                         | 6 ++++++
+ tink/controllers/workflow/controller.go        | 4 ++--
+ 8 files changed, 15 insertions(+), 7 deletions(-)
+
+diff --git a/config/crd/bases/tinkerbell.org_hardware.yaml b/config/crd/bases/tinkerbell.org_hardware.yaml
+index 47f5edb..e0a52a1 100644
+--- a/config/crd/bases/tinkerbell.org_hardware.yaml
++++ b/config/crd/bases/tinkerbell.org_hardware.yaml
+@@ -16,7 +16,7 @@ spec:
+     listKind: HardwareList
+     plural: hardware
+     singular: hardware
+-  scope: Cluster
++  scope: Namespaced
+   versions:
+   - name: v1alpha1
+     schema:
+diff --git a/config/crd/bases/tinkerbell.org_templates.yaml b/config/crd/bases/tinkerbell.org_templates.yaml
+index 4e0a558..c364dab 100644
+--- a/config/crd/bases/tinkerbell.org_templates.yaml
++++ b/config/crd/bases/tinkerbell.org_templates.yaml
+@@ -16,7 +16,7 @@ spec:
+     listKind: TemplateList
+     plural: templates
+     singular: template
+-  scope: Cluster
++  scope: Namespaced
+   versions:
+   - name: v1alpha1
+     schema:
+diff --git a/config/crd/bases/tinkerbell.org_workflows.yaml b/config/crd/bases/tinkerbell.org_workflows.yaml
+index e9a4d5a..11f871d 100644
+--- a/config/crd/bases/tinkerbell.org_workflows.yaml
++++ b/config/crd/bases/tinkerbell.org_workflows.yaml
+@@ -16,7 +16,7 @@ spec:
+     listKind: WorkflowList
+     plural: workflows
+     singular: workflow
+-  scope: Cluster
++  scope: Namespaced
+   versions:
+   - name: v1alpha1
+     schema:
+diff --git a/config/default/manager_image_patch.yaml b/config/default/manager_image_patch.yaml
+index af4861a..2b3b15c 100644
+--- a/config/default/manager_image_patch.yaml
++++ b/config/default/manager_image_patch.yaml
```

Comment:
I think this file can be skipped. This change comes from `make release` and I dont think we need it permanently.